packages/dashboards/addon/components/dashboard-action-list.hbs

```diff
@@ -1,82 +1,91 @@
-            id="navi-dashboard-action-schedule-{{@index}}"
-            class="schedule"
-            @model={{dashboard}}
-            @disabled={{not dashboard.validations.isTruelyValid}}
-            @onSave={{delivery-rule-action "SAVE_DELIVERY_RULE"}}
-            @onRevert={{delivery-rule-action "REVERT_DELIVERY_RULE"}}
-            @onDelete={{delivery-rule-action "DELETE_DELIVERY_RULE"}}
-            as |onOpen|
-          >
-            <button 
-              type="button"
-              id="navi-dashboard-action-schedule-{{@index}}"
-              disabled={{not dashboard.validations.isTruelyValid}}
-              class="schedule-action__button"
-              {{on "click" onOpen}}
-            >
-              <NaviIcon @icon="clock-o" class="navi-icon__schedule" />
-            </button>
-            <EmberTooltip @targetId="navi-dashboard-action-schedule-{{@index}}">
-              {{if dashboard.validations.isTruelyValid "Schedule the dashboard" "Validate dashboard to enable scheduling"}}
-            </EmberTooltip>
-          </DashboardActions::Schedule>
-        </li>
-      {{/if}}
-
-      <li class="action">
-        <CommonActions::Delete
-          class="delete"
+        <DashboardActions::Schedule
+          class="schedule"
           @model={{dashboard}}
-          @warnMsg="Are you sure you want to delete this dashboard and all of its widgets?"
-          @deleteAction={{item-action "DELETE_ITEM" dashboard}}
+          @disabled={{not dashboard.validations.isTruelyValid}}
+          @onSave={{delivery-rule-action "SAVE_DELIVERY_RULE"}}
+          @onRevert={{delivery-rule-action "REVERT_DELIVERY_RULE"}}
+          @onDelete={{delivery-rule-action "DELETE_DELIVERY_RULE"}}
+          as |onOpen|
         >
-          <NaviIcon @icon="trash-o" class="navi-icon__delete" />
-          <EmberTooltip @text="Delete the dashboard"/>
-        </CommonActions::Delete>
-      </li>
+          <DenaliIcon 
+            id="navi-dashboard-action-schedule-{{@index}}"
+            disabled={{not dashboard.validations.isTruelyValid}}
+            @icon="time"
+            @size="small"
+            class="navi-icon__schedule schedule-action__button p-4 link"
+            {{on "click" onOpen}}
+          />
+          <EmberTooltip @targetId="navi-dashboard-action-schedule-{{@index}}">
+            {{if dashboard.validations.isTruelyValid "Schedule the dashboard" "Validate dashboard to enable scheduling"}}
+          </EmberTooltip>
+        </DashboardActions::Schedule>
+      {{/if}}
+      <CommonActions::Delete
+        @model={{dashboard}}
+        @warnMsg="Are you sure you want to delete this dashboard and all of its widgets?"
+        @deleteAction={{item-action "DELETE_ITEM" dashboard}}
+        as |onDelete|
+      >
+        <DenaliIcon 
+          id="navi-dashboard-action-delete-{{@index}}"
+          @size="small" 
+          @icon="trash" 
+          class="navi-icon__delete delete p-4 link" 
```

Comment:
Can we bemify this? Maybe `dashboard-actions__delete` (I guess that would we name the component to `dashboard-actions`)